docs/design/expressions/README.md

```diff
@@ -345,6 +337,23 @@ The binary arithmetic and bitwise operators also have
 [compound assignment](/docs/design/assignment.md) forms. These are statements
 rather than expressions, and do not produce a value.
 
+## Suffix ops
```

Comment:
"ops" here is a bit ambiguous -- is this "operators" or "operations"? (Or is that ambiguity intentional?) I think this is clear, but avoiding the abbreviation might be better.